glusterfsd/src/glusterfsd.c

```diff
@@ -276,6 +276,8 @@ static struct argp_option gf_options[] = {
     {"fuse-dev-eperm-ratelimit-ns", ARGP_FUSE_DEV_EPERM_RATELIMIT_NS_KEY,
      "OPTIONS", OPTION_HIDDEN,
      "rate limit reading from fuse device upon EPERM failure"},
+    {"filesystem-name", ARGP_FUSE_FSNAME_KEY, "OPTIONS", OPTION_HIDDEN,
```

Comment:
Not sure whether `display-name` could be a better name for this option.

Reply:
picking `fs-display-name`